license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -201,6 +224,10 @@
       "bundleName": "apache2",
       "licenseNamePattern": "^Apache License,? 2\\.0$"
     },
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^Apache Software License$"
```

Comment:
question: Are we sure that this is Apache 2.0? The license name looks more like an older version of Apache. Can we check license text?

Reply:
"cyclonedx-python-lib","7.5.1","Apache Software License" - output in pip-licenses

https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/LICENSE - license itself, in title Apache Version 2.0